immudb/grpc/proto/schema.proto

```diff
@@ -328,38 +369,90 @@ message Database {
 	string databaseName = 1;
 }
 
+message DatabaseSettings {
+	string databaseName = 1;
+	bool   replica = 2;
+	string masterDatabase = 3;
+	string masterAddress = 4;
+	uint32 masterPort = 5;
+	string followerUsername = 6;
+	string followerPassword = 7;
+
+	uint32 fileSize = 8;
+	uint32 maxKeyLen = 9;
+	uint32 maxValueLen = 10;
+	uint32 maxTxEntries = 11;
+
+	bool   excludeCommitTime = 12;
+
+	uint32 maxConcurrency = 13;
+	uint32 maxIOConcurrency = 14;
+
+	uint32 txLogCacheSize = 15;
+	uint32 vLogMaxOpenedFiles = 16;
+	uint32 txLogMaxOpenedFiles = 17;
+	uint32 commitLogMaxOpenedFiles = 18;
+
+	IndexSettings indexSettings = 19;
+}
+
+message IndexSettings {
```

Comment:
This message was added after 1.2.2 and is now changed in current master - I think that we should use proto definition from 1.2.2 to avoid introducing breaking changes in the API.